specification/digitaltwins/data-plane/Microsoft.DigitalTwins/stable/2022-05-31/digitaltwins.json

```diff
@@ -0,0 +1,1622 @@
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IncomingRelationship"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "A URI to retrieve the next page of objects."
+        }
+      }
+    },
+    "IncomingRelationship": {
+      "type": "object",
+      "description": "An incoming relationship.",
+      "properties": {
+        "$relationshipId": {
+          "type": "string",
+          "description": "A user-provided string representing the id of this relationship, unique in the context of the source digital twin, i.e. sourceId + relationshipId is unique in the context of the service."
+        },
+        "$sourceId": {
+          "type": "string",
+          "description": "The id of the source digital twin."
+        },
+        "$relationshipName": {
+          "type": "string",
+          "description": "The name of the relationship."
+        },
+        "$relationshipLink": {
+          "type": "string",
+          "description": "Link to the relationship, to be used for deletion."
+        }
+      }
+    },
+    "QuerySpecification": {
+      "description": "A query specification containing either a query statement or a continuation token from a previous query result.",
+      "type": "object",
+      "properties": {
+        "query": {
+          "description": "The query to execute. This value is ignored if a continuation token is provided.",
+          "type": "string"
+        },
+        "continuationToken": {
+          "description": "A token which is used to retrieve the next set of results from a previous query.",
+          "type": "string"
+        }
+      }
+    },
+    "QueryResult": {
+      "description": "The results of a query operation and an optional continuation token.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The query results.",
+          "type": "array",
+          "items": {
+            "type": "object"
+          }
+        },
+        "continuationToken": {
+          "description": "A token which can be used to construct a new QuerySpecification to retrieve the next set of results.",
+          "type": "string"
+        }
+      }
+    },
+    "ErrorResponse": {
+      "description": "Error response.",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/Error",
+          "description": "The error details."
+        }
+      }
+    },
+    "Error": {
+      "description": "Error definition.",
+      "properties": {
+        "code": {
+          "description": "Service specific error code which serves as the substatus for the HTTP error code.",
+          "type": "string",
+          "readOnly": true
+        },
+        "message": {
+          "description": "A human-readable representation of the error.",
+          "type": "string",
+          "readOnly": true
+        },
```

Comment:
nit, error and code and message should be required.

Reply:
Done